package.json

```diff
@@ -22,7 +23,8 @@
         "react": "^16.8.6",
         "react-dom": "^16.8.6",
         "react-intl": "^2.9.0",
-        "react-select": "^2.0.0"
+        "react-select": "^2.0.0",
+        "typescript": "^3.7.3"
```

Comment:
This peer dependency will currently not work with the DE.  We could go with a branching strategy like you suggested, but then I'm not sure how that will work since we push and build `ui-lib` as a version to npm 🤔 I'm not sure if we want to just version lock the DE to whatever version it's at now and let Sonora get all the updates?

Reply:
Yeah, I think we can limit the DE to the current major version (allowing for bug fixes as minor version updates), but let the Sonora get the latest updates. Honestly, I think that's going to be required.

Reply:
For clarification, I mean something like limit the DE to `^1.0.0` (just as an example) which is built and pushed to npm from the `DE` branch, while master continues to have the lastest code starting with version `2.0.0`. I pulled those version numbers out of the air just as examples.